### DISCUSSION TITLE
Search is not good for View Transitions

### DISCUSSION THREAD

**Discussion by louiss0** [shishkin/astro-pagefind#63] [General]
The search component will not work when someone decides to use View Transitions. 
The reason is because Astro will persist current state of each script throughout each transition.
Astro doesn't re-execute scripts throughout each transition instead just saves each executed script.

**Comment by shishkin** [comment]
Support for view transitions has been added in [v1.4.0](https://github.com/shishkin/astro-pagefind/releases/tag/v1.4.0)

**Reply by louiss0** [reply]
> Support for view transitions has been added in [v1.4.0](https://github.com/shishkin/astro-pagefind/releases/tag/v1.4.0)

Thanks now I can use it!
